tests/scripts/mbedtls_test_dev/__init__.py

```diff
@@ -0,0 +1,7 @@
+import os
+import sys
+
```

Comment:
> Unfortunately we cannot re-use the same name and if we would like to contain python files in multiple locations and expose them as a single pacakge we should consider [namespace packages](https://peps.python.org/pep-0420/#abstract)
>
> I think it is considerably more complicated, then just using the proposed trick with all imports.

What's complicated about namespace packages? If I'm reading the PEP right, they're available natively since Python 3.3, and we currently require ≥3.5 for our CI and ≥3.8 as the documented minimum for Mbed TLS 3.6 LTS. And as far as I can tell, all they require is a bit of ugliness in `__init__.py` files, which isn't worse than our current ugliness.

Reply:
Complicated as if people wil need to understand it since it's not very common on small projects.

The python guidance is:

-  if you want keep all the library stuff under one directory -> Use packages with `__init__` magic.
- If you want to expose a single package that uses code in multiple directories --> Namespace packages were created for that reason.

Reply:
> If namespaces give us some flexibility, it doesn't matter how complicated they are under the hood. Are they complicated for people who will create new modules, create scripts that use modules, and run checkers (pylint/mypy)?

That unfortunately I do not know. I never had to reach that level of complexity in a project I was working on. But I believe it shouldn't be an issue for people creating modules or scripts, as long as they are placed in the right location ( you set the folder structure and the contents are sorted accordingly )

For pylint/mympy I suspect there coud be issues [1](https://stackoverflow.com/questions/48668176/mypy-and-distributing-a-namespace-package) since those tools focus on testing simple use-cases. But because namespaces exist for a while we could be ok by updating the version we use. e.g [2](https://github.com/python/mypy/issues/5759)

The best way to go about it is trying to create a mock namespace, and then see it works with our enviroment.